Dockerfile

```diff
@@ -7,6 +7,8 @@ FROM ubuntu:20.04
 
 ########################## BEGIN INSTALLATION #########################
 
+ENV NUM_CPUS 1
```

Comment:
This syntax is deprecated, please use ENV NAME=VALUE syntax as per [the docs](https://docs.docker.com/engine/reference/builder/#env)

Reply:
I just remembered you'd given me edit access, I have fixed these.